### DISCUSSION TITLE
Colors in cargo flow

### DISCUSSION THREAD

**Discussion by freddepettersson** [tellurianinteractive/Tellurian.Trains.ModulesRegistryApp#67] [Q&A] [answered]
Hi
I added new cargo , but i can still select same track but color do now follow, I can select difrent colors as show in picture.

![image](https://user-images.githubusercontent.com/80942768/115057098-ed3e9b00-9ee3-11eb-8d27-db8f5f624221.png)

Are the color ment to be used as Fremo YP ?
I think it would be more logic to select colour in the stationcustomer header to be general for all added cargo flow rows,( top of picture).
When adding diffrent cargo to the track /loading spot , the color and track already are selected in the header of the track.
If it needed for database , then copy header and hide this rows in the cargo flow rows 

/Fredrik

**Comment by fjallemark** [comment, marked as answer]
The colors are *not* ment to be origin and destination colors, but are intended to improve orientation within the station.
The customer's *track or loading area* and *colour* will be default for all *cargo flows* unless you specify a deviating *track or loading area* and *colour* for a specific *cargo flow*. If you want the default to apply, simply leave the *track or loading area* empty for the *cargo flow'.
